CHANGELOG.md

```diff
@@ -0,0 +1,66 @@
+# Changelog
+
+All notable changes to the Solar Forecast Arbiter Framework will be documented
+in this file.
+
+Detailed changes to the Solar Forecast Arbiter Core python library can be found
+in the core documentation's [what's new](https://solarforecastarbiter-core.readthedocs.io/en/latest/whatsnew/index.html) series.
+
+## [Unreleased]
+
+### Added
+
+- Report form now includes all deterministic metrics options identified by
+  stakeholders. Also includes options to calculate metrics by categories Total,
+  Year, Month, Date, and Hour of Day.
+
+- Ability to compare Forecasts to Aggregates in reports.
+
+- Report's may be downloaded in HTML format at
+  `/reports/<report_id>/downloads/html`. Note that there are no currently
+  functioning links to this endpoint.
+
+- Report downloads contain a GPG signed report as well as md5, sha1 and sha256
+  checksums for validation.
+
+- CHANGELOG.md (this file) for tracking and communicating changes.
+
+
```

Comment:
is https://github.com/SolarArbiter/solarforecastarbiter-api/commit/8f66fe45beda692dd4d4d92bf82c6f5b14173051 worth adding?

Reply:
sure